formal-models/README.md

```diff
@@ -112,6 +112,215 @@ configuration:
 * [TLA⁺ specification](./dbftMultipool/dbftMultipool.tla)
 * [TLC Model Checker configuration](./dbftMultipool/dbftMultipool___AllGoodModel.launch)
 
+## Proposed dBFT 2.1 models
+
+Based on the liveness locks scenarios found by the TLC model checker in the
+[basic dBFT 2.0 model](#basic-dbft-20-model) we've developed two extensions of
+dBFT 2.0 protocol that allow to avoid the liveness lock problem and to preserve
+the safety properties of the algorithm. The extensions currently doesn't have
+the code-level implementation and presented as a TLA⁺ specifications ready to be
+reviewed and discussed. The improved protocol presented in the extensions will
+be referred below as dBFT 2.1.
+
+We've checked both dBFT 2.1 models with the TLC Model Checker against the same
+set of launch configurations that was used to reveal the liveness problems of the
+[basic dBFT 2.0 model](#basic-dbft-20-model). The improved models have larger
+set of states, thus, the TLC Model Checker wasn't able to finish the liveness
+requirements checks for *all* possible states. However, the checks have passed for
+a state graph diameter that was large enough to believe the presented models
+solve the dBFT 2.0 liveness lock problems.
+
+### dBFT 2.1 stages-based model
```

Comment:
@roman-khimov, could you, please, review this part (`dBFT 2.1 stages-based model`) more carefully? We probably may want to add more details about the reason of adding some of the transitions between node states.